### PR TITLE
Added .nvmrc file with the desired node version for those using nvm.

### PR DESCRIPTION
**Pull request by ifeins**
Also fixes this issue - https://github.com/eshaham/israeli-ynab-updater/issues/9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,30 +4,37 @@ A tool for updating YNAB using israeli-bank-scrapers
...
+If you're using `nodenv`, it should automatically pick up the correct node version.
+
 Next you will need to install dependencies by running
 ```bash
 npm install
 ```
 
 ### Saving credentials
-The credentials for each scraper are encrypted and saved in an dedicated file on your system. 
+The credentials for each scraper are encrypted and saved in an dedicated file on your system.<br/>
```
Comment (eshaham) [inline README.md]: @ifeins no need for `<br/>` in markdown 😄, can you remove it?

Comment (ifeins) [reply README.md]: How do you add a newline in markdown?
I think it's much more readable with newlines, but I don't feel strongly about it.

Compare the [README in this branch](https://github.com/ifeins/israeli-ynab-updater/blob/add_nvmrc/README.md) to the [README in master](https://github.com/eshaham/israeli-ynab-updater/blob/master/README.md)

Comment (eshaham) [reply README.md]: @ifeins you can use 2 spaces at the end of the first line instead, which will also generate a line break.

Comment (ifeins) [reply README.md]: @eshaham Cool, didn't know that, I've updated the branch with these changes.

**Pull request merged**